vagrant/bootstrap.sh

```diff
@@ -52,7 +52,7 @@ apt-get install -y \
   python-pip python-dev libffi-dev redis-server python-plaso plaso-tools jq
 
 # Install python dependencies
-pip install --upgrade pip
+# pip -v install --upgrade pip  # don't do this https://github.com/pypa/pip/issues/5221
```

Comment:
Looks good. just remove the line altogether IMO.

Reply:
Passing `--user` as well to pip, is another option here. :)